druid/src/widget/widget_ext.rs

```diff
@@ -234,8 +234,8 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
     /// Wrap this widget in a [`LensWrap`] widget for the provided [`Lens`].
     ///
     ///
-    /// [`LensWrap`]: struct.LensWrap.html
-    /// [`Lens`]: trait.Lens.html
+    /// [`LensWrap`]: crate::widget::LensWrap
+    /// [`Lens`]: crate::Lens
```

Comment:
you can remove these lines btw, because `LensWrap` and `Lens` are already in scope!